jms/src/main/java/com/sun/ts/tests/jms/core/bytesMsgQueue/BytesMsgQueueTestsJspTest.java

```diff
@@ -75,27 +75,27 @@ public static EnterpriseArchive createDeploymentVehicle(@ArquillianResource Test
             com.sun.ts.tests.common.vehicle.VehicleClient.class
             );
             // The web.xml descriptor
-            URL warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.xml");
+            URL warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/jms/core/bytesMsgQueue/jsp_vehicle_web.xml");
             if(warResURL != null) {
               bytesMsgQueue_jsp_vehicle_web.addAsWebInfResource(warResURL, "web.xml");
             }
             // The sun-web.xml descriptor
-            warResURL = BytesMsgQueueTests.class.getResource("//com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.war.sun-web.xml");
+            warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/jms/core/bytesMsgQueue/bytesMsgQueue_jsp_vehicle_web.war.sun-web.xml");
             if(warResURL != null) {
               bytesMsgQueue_jsp_vehicle_web.addAsWebInfResource(warResURL, "sun-web.xml");
             }
 
             // Any libraries added to the war
 
             // Web content
-            warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.xml");
-            if(warResURL != null) {
-              bytesMsgQueue_jsp_vehicle_web.addAsWebResource(warResURL, "/WEB-INF/jsp_vehicle_web.xml");
-            }
-            warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/jms/core/bytesMsgQueue/jsp_vehicle_web.xml");
-            if(warResURL != null) {
-              bytesMsgQueue_jsp_vehicle_web.addAsWebResource(warResURL, "/WEB-INF/jsp_vehicle_web.xml");
-            }
+            // warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.xml");
```

Comment:
@starksm64 Are these required?

Reply:
The generated code looks in several places. Any test local descriptor should be preferred, but there is no jsp_vehicle_web.xml in jms/src/main/java/com/sun/ts/tests/jms/core/bytesMsgQueue, so this code block that should load the common/src/main/java/com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.xml version is what should be correct for this test:
```
warResURL = BytesMsgQueueTests.class.getResource("/com/sun/ts/tests/common/vehicle/jsp/jsp_vehicle_web.xml");
            if(warResURL != null) {
              bytesMsgQueue_jsp_vehicle_web.addAsWebResource(warResURL, "/WEB-INF/jsp_vehicle_web.xml");
            }
```

Reply:
There is a local jms/src/main/java/com/sun/ts/tests/jms/core/bytesMsgQueue/jsp_vehicle_web.xml which should be added as WEB-INF/web.xml (at line 78). So this code block effectively adds the same file as /WEB-INF/jsp_vehicle_web.xml ?

Reply:
No, the code is looking in various places. Only one will be needed.